### DISCUSSION TITLE
Is the Goal of Remix Auth to be similar to Next-Auth but for the Remix ecosystem?  📀🔐

### DISCUSSION THREAD

**Discussion by cliffordfajardo** [sergiodxa/remix-auth#85] [General]
Hi @sergiodxa,

In the READMe you mentioned that remix-auth is inspired by passportjs, which I think is great!

Does this project aspire to be something similar to NextJS Auth, where it might support multiple provider options?
- https://github.com/nextauthjs/next-auth#add-api-route

For reference, I think NextJS Auth and [Notifire](https://github.com/notifirehq/notifire)'s API's are very similar and are great - thanks @balazsorban44, @iaincollins!

Is the goal of Remix Auth to be similar in Spirit to those (support multiple providers) or do you think it will be limited in scope?

**Comment by sergiodxa** [comment]
That lib is like Passport for Next, so yeah, but I liked more the API of Passport that’s why it’s inspired by it.

And while for now all the supported strategies come from the lib itself like in NextAuth.js it doesn’t need to work that way, as with Passport you should be able to publish your strategies individually (in a v3 I may move the strategies to another package too)
